src/storage/vpid_utilities.hpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#ifndef _VPID_UTILS_HPP_
+#define _VPID_UTILS_HPP_
+
+#include <cstdint>
+#include <cstdio>
+
+struct vpid;
+namespace cubpacking
+{
+  class packer;
+  class unpacker;
+}
+
+namespace vpid_utils
+{
+  void pack (const vpid &vpd, cubpacking::packer &serializer);
+  void unpack (cubpacking::unpacker &deserializer, vpid &vpd);
+  size_t get_packed_size (cubpacking::packer &serializator, std::size_t start_offset);
```

Comment:
Let's give the function [constexpr ](https://en.cppreference.com/w/cpp/language/constexpr) specifier. packer::get_packed_bigint_size has to get constexpr too.
```suggestion
  constexpr size_t get_packed_size (cubpacking::packer &serializator, std::size_t start_offset);
```